xAPI.md

```diff
@@ -544,9 +552,22 @@ __Verb__: Defines the action being done by the Actor within the Activity within
 
 <a name="xapi-components" />
 
-## 5.0 xAPI Components
+<a name="xapi_ecosystem"/>
+## 5.0 The xAPI Ecosystem
```

Comment:
I suggested renaming the section 'xAPI Ecosystem' because I believe that in an introduction, one should try to provide a broad overview of the system being discussed - both from a structural and a functional point of view.  For me, 'xAPI Components' suggests more of a parts list rather than an overview of how these main components are structured and what their broad functions are.  I had thought of suggesting 'xAPI System' - but I think that it could too easily be perceived by readers that we are discussing a computer system - whereas the general perception of an ecosystem is that it is a broad, complex system which includes a wide variety of different components/organisms.  I thought it was also useful to include the Communities of Practice as a subsection to because they provide a critical function of providing shared meanings to different communities that are using the specification.

I'd be interested in hearing why you prefer the term 'xAPI Components' @garemoko.